app/models/managed_interactive.rb

```diff
@@ -133,7 +133,8 @@ def to_hash
       inherit_click_to_play_prompt: inherit_click_to_play_prompt,
       custom_click_to_play_prompt: custom_click_to_play_prompt,
       inherit_image_url: inherit_image_url,
-      custom_image_url: custom_image_url
+      custom_image_url: custom_image_url,
+      linked_interactives: linked_interactives_list
```

Comment:
If you need to remove the linked_interactives on the duplicate call, then you probably need to remove them from the export too. This is because export and import is basically the same as duplicate.

But I also wonder about this approach. The AP must know about the linked_interactive and it is based on the export, which I would assume uses to_hash. Perhaps it uses to_interactive?  

Could you add a comment above `to_hash` and `to_interactive` which describes what uses each of them, and how they are different?  It seems we keep running into problems about which one to use where. I no longer remember why we have both of them.

Reply:
Thanks. Removing `linked_interactives` from the export got the breaking tests to pass.

I added comments about `to_hash` and `to_interactive` though I'm not sure we want to keep them exactly as is.

Would it be worth investigating using `to_authoring_hash` (in app/models/base_interactive.rb) instead of `to_hash` for managed and MW interactives? I see it includes `linked_interactives` too. I think we may have briefly noted that, but then didn't try using it, but I can't recall why.